benchmarks/cpu/lfm2_parallel_linear.cpp

```
@@ -0,0 +1,295 @@
+  state = state * 1664525U + 1013904223U;
+  return state;
+}
+
+float deterministicValue(uint32_t& state) {
+  const int32_t centered = static_cast<int32_t>((nextRandom(state) >> 8U) % 2001U) - 1000;
+  return static_cast<float>(centered) / 4096.0F;
+}
+
+int parsePositiveInt(const char* value, const char* name) {
+  char* end = nullptr;
+  const long parsed = std::strtol(value, &end, 10);
```

Clang (14.0.6)

[error] consider replacing 'long' with 'int64'

(google-runtime-int,-warnings-as-errors)